Cargo.toml

```diff
@@ -9,6 +9,7 @@ members = [
     "examples/demo-simple-stf",
     "examples/demo-rollup",
     "examples/demo-nft-module",
+    "examples/simple-nft-module",
```

Comment:
Is the module  generic enough to be considered as a standard module? If yes then let's put it in `module-implementations`

Reply:
moved it to "module-implementations/sov-nft-module"